### ISSUE TITLE
Topic Information Interface

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#45]
## User story
1. As a frontend dev
2. I need an interface
3. Which I can use to retrieve data about the topics

## Acceptance criteria
* [ ] Interface that provides all information mentioned in the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) about the topics

(optional) Further propositions of information that could be relevant to the user should be mentioned in comments down below

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
